src/Symfony/Component/Notifier/Bridge/Slack/SlackTransport.php

```diff
@@ -21,30 +21,30 @@
 
 /**
  * @author Fabien Potencier <fabien@symfony.com>
+ * @author Daniel Stancu <birkof@birkof.ro>
  *
  * @internal
  *
- * @experimental in 5.0
+ * @experimental in 5.1
  */
 final class SlackTransport extends AbstractTransport
 {
-    protected const HOST = 'slack.com';
+    protected const HOST = 'hooks.slack.com';
 
-    private $accessToken;
-    private $chatChannel;
+    protected $client;
```

Comment:
It should be private. 

It is a final class so nobody can use a `$client` anyways.

Reply:
Fixed!